lightning/src/routing/router.rs

```diff
@@ -3831,6 +3852,64 @@ mod tests {
 		}
 	}
 
+	#[test]
+	fn total_fees_single_path() {
+		let route = Route {
+			paths: vec![vec![
+				RouteHop {
+					pubkey: PublicKey::from_slice(&hex::decode("02eec7245d6b7d2ccb30380bfbe2a3648cd7a942653f5aa340edcea1f283686619").unwrap()[..]).unwrap(),
+					channel_features: ChannelFeatures::empty(), node_features: NodeFeatures::empty(),
+					short_channel_id: 0, fee_msat: 100, cltv_expiry_delta: 0 // Test vectors are garbage and not generateble from a RouteHop, we fill in payloads manually
```

Comment:
nit here and below: s/generateble/generatable (i think these comments can all be deleted though :stuck_out_tongue:)